pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,105 @@
-schema = 1
-
-[device]
-name = "lightning.kokkos"
-
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
+schema = 2
 
 # The union of all gate types listed in this section must match what
 # the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "Identity",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "S",
-        "T",
-        "PhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CNOT",
-        "CY",
-        "CZ",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "IsingXX",
-        "IsingXY",
-        "IsingYY",
-        "IsingZZ",
-        "ControlledPhaseShift",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "SingleExcitation",
-        "SingleExcitationPlus",
-        "SingleExcitationMinus",
-        "DoubleExcitation",
-        "DoubleExcitationPlus",
-        "DoubleExcitationMinus",
-        "MultiRZ",
-        "QubitUnitary",
-        "GlobalPhase",
-]
+[operators.gates.native]
+
+CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
```

Comment:
It may be just a matter of convention, but do we want to mark non-parametric gates differentiable? One issue which might come up is that if one reads that and tries to apply Lightning's adjoint on CNOT, it will fail because CNOT has no generator (instead of an all-zero generator).

Reply:
I suppose it is differentiable in the sense that it is allowed in adjoint Jacobian circuit. The purpose is to identify gates that need to be decomposed because they are not allowed in an adjoint Jacobian circuit 🤔

Reply:
@vincentmr if we follow the convention saying that this file describes device APIs (to which I agree), and, specifically, that the `gates.native` section describes the C++ API of the device, then I think we should not assume direct interpretation by a user.

Reply:
I agree with Vincent that it is mainly a matter of convention, but one advantage I see with attaching the "differentiable" tag to non-parametrized gates is that we don't need any additional information on whether to decompose the gate inside an adjoint-Jacobian circuit. With the other scheme we would additionally need to query the gate for whether it has differentiable parameters, and only if it does should we check for the "differentiable" tag in the TOML. But I'm open to go with that solution as well if it offers other benefits.

Reply:
I don't mind either way, do as you see fit.